e2e/test/scenarios/filters-reproductions/dashboard-filters-reproductions.cy.spec.js

```diff
@@ -1782,14 +1782,22 @@ describe("issue 25374", () => {
   };
 
   beforeEach(() => {
-    cy.intercept("POST", "/api/card/*/query").as("cardQuery");
-    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card//*/query").as(
-      "dashcardQuery",
-    );
+    // we are overriding the intercept set in e2e/support/helpers/e2e-misc-helpers.js
+    cy.intercept(
```

Comment:
afaik we add a new intercept here, there is no way to override old intercepts, but only last matching intercept will actually handle request, so idk if "overriding" is a correct word here